src/components/integrated/OakQuizTextInput/OakQuizTextInput.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
```

Comment:
@kimon-satan I'm conflicted as to whether this belongs in `ui`, `integrated` or maybe even `base` within the current structure.

Going by atomic design this would probably be classed as an "atom". But i'm given pause by this line in the integrated component docs:

> Applicable to a limited range of contexts which might occur in multiple apps 

This is a pretty specialised flavour of input, but it isn't overly complex and I wouldn't put it in the "molecule" box of atomic design.

To go with the Lego analogy, I see `OakTextInput` as a "brick" but `OakQuizTextInput` is some flavour of ["brick modified"](https://www.bricklink.com/v2/catalog/catalogitem.page?P=2877#T=C) — still a brick, but with a more specific purpose.

If I gave this component responsibility for the field label and brought in `OakLabel` it would probably more comfortably fit into the "molecule" camp.

Interested in your take 🙇🏼

Reply:
Yeah I can see the problem.  Lets consider the folder rules like a ratchet - start at the bottom of the hierarchy if a rule fails move up a level. In this case it fails the context rules in base and ui so it should belong in integrated even though it doesn't pass all the tests there . Alternatively just view it as this ;)

![image](https://github.com/oaknational/oak-components/assets/1030540/878e5ea8-2651-4d37-8ebc-f941e79f630f)